x-pack/metricbeat/module/meraki/device_health/device_network_health_channel_utilization.go

```diff
@@ -0,0 +1,96 @@
+package device_health
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+	meraki_api "github.com/meraki/dashboard-api-go/v3/sdk"
+)
+
+func getNetworkHealthChannelUtilization(client *meraki_api.Client, networks *meraki_api.ResponseOrganizationsGetOrganizationNetworks) ([]*meraki_api.ResponseNetworksGetNetworkNetworkHealthChannelUtilization, error) {
+
+	var networkHealthUtilizations []*meraki_api.ResponseNetworksGetNetworkNetworkHealthChannelUtilization
+
+	for _, network := range *networks {
+		for _, product_type := range network.ProductTypes {
+			if product_type == "wireless" {
+				networkHealthUtilization, res, err := client.Networks.GetNetworkNetworkHealthChannelUtilization(network.ID, &meraki_api.GetNetworkNetworkHealthChannelUtilizationQueryParams{})
```

Comment:
we are still pulling one days worth of data each time we run this - we should only pull data for the current collection period. 

we talked offline a little about simplifying things here too, to ensure we only ever get one bucket per call (by specifying a maximum collection period no greater than the resolution of these metrics), were you able to try it out to verify it behaves as expected?

Reply:
Before I resolved comment, we were looking for additional input, on comments,  I believe you had @ asked someone and they never got back to us ... I was looking for guidance on 10 minutes or 60 minutes, since no input, I left it at defaults.   I can switch to static 3600 seconds (1 hr) if you want so it matches the wireless default.  ???